### PR TITLE
feat: add log level config

### PR DESCRIPTION
**Pull request by craigpastro**
## Description

Add log level config: `none`, `debug`, or `info`.

## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

**Comment by jon-whit** [comment]
We should support all levels of the underlying logger interface (`debug`, `info`, `warn`, `error`, `panic`, `fatal`)

File: cmd/run.go [hunk 1]
```diff
@@ -301,6 +305,14 @@ func VerifyConfig(cfg *Config) error {
 		return fmt.Errorf("config 'http.upstreamTimeout' (%s) cannot be lower than 'listObjectsDeadline' config (%s)", cfg.HTTP.UpstreamTimeout, cfg.ListObjectsDeadline)
 	}
 
+	if cfg.Log.Format != "text" && cfg.Log.Format != "json" {
+		return fmt.Errorf("log format must be either 'text' or 'json'")
```
Comment (jon-whit) [inline cmd/run.go:309]: config 'log.format' must be one of ['text' or 'json']

File: cmd/run.go [hunk 2]
```diff
@@ -787,6 +786,9 @@ func bindRunFlags(cmd *cobra.Command) {
 	cmd.Flags().String("log-format", defaultConfig.Log.Format, "the log format to output logs in")
 	util.MustBindPFlag("log.format", cmd.Flags().Lookup("log-format"))
 
+	cmd.Flags().String("log-level", defaultConfig.Log.Level, "the log level to use")
```
Comment (jon-whit) [inline cmd/run.go:789]: Mind documenting this in the `.config-schema.json` file as well?

File: cmd/run.go (hunk 1 above)
Comment (craigpastro) [reply cmd/run.go:309]: Updated. Thanks!

File: cmd/run.go (hunk 2 above)
Comment (craigpastro) [reply cmd/run.go:789]: Always forget that. I wish we simply stuck with env vars 😢

File: .config-schema.json [hunk 3]
```diff
@@ -184,6 +184,12 @@
                     "type": "string",
                     "enum": ["text", "json"],
                     "default": "text"
+                },
+                "level": {
+                    "description": "The log level to set. For production we recommend 'info' format.",
```
Comment (jon-whit) [inline .config-schema.json:189]: @craigpastro would you please submit a PR to openfga.dev and add this to the Production Checklist?

**Review by jon-whit — APPROVED** [review]

File: .config-schema.json (hunk 3 above)
Comment (craigpastro) [reply .config-schema.json:189]: Sure. Thanks!

**Pull request merged**